### PR TITLE
Feature/remove email phone userprefs

### PR DESCRIPTION
**Pull request by DarinHajou**
Allowed email and phone_number to be nullable in the UserPreferences schema, database, and service layers

### CODE REVIEW

File: src/models/UserSettings.ts [hunk 1]
```diff
@@ -18,21 +17,21 @@ const userSettingsSchema = new Schema<IUserSettings>(
...
     image: {
       type: String,
       required: false,
-      default: ''
     },
     findme: {
       type: String,
-      enum: Object.values(DeviceLocationType).filter(value => typeof value === 'string'),
       required: true,
-      default: DeviceLocationType.Automatic
```
Comment (swoocn) [inline src/models/UserSettings.ts:35]: Is this a mistake?  Why is this changed..? 🤔

Comment (DarinHajou) [reply src/models/UserSettings.ts:35]: I was going to mention that I got some odd merge conflicts with this branch. Probably bcos I forgot to pull.

Comment (DarinHajou) [reply src/models/UserSettings.ts:35]: Have an closer look at it before merge. Alternatively, I could redo it quickly on a mew branch

**Comment by swoocn** [comment]
Hi @DarinHajou, I've noticed that recent changes from shared dev seem to be missing in your branch.  I have a hunch as to what might be happening based on your merge history.  Just to check, are you you running  `git merge origin/dev` rather than `git merge dev`?  The latter merges your local dev branch, so it will only have the changes from your last local pull.  The first command is the correct approach to sync with the latest updates from the shared/ remote dev branch.  Let me know if you have any questions and I will be happy to explain.

**Comment by DarinHajou** [comment]
> Hi @DarinHajou, I've noticed that recent changes from shared dev seem to be missing in your branch. I have a hunch as to what might be happening based on your merge history. Just to check, are you you running `git merge origin/dev` rather than `git merge dev`? The latter merges your local dev branch, so it will only have the changes from your last local pull. The first command is the correct approach to sync with the latest updates from the shared/ remote dev branch. Let me know if you have any questions and I will be happy to explain.



> Hi @DarinHajou, I've noticed that recent changes from shared dev seem to be missing in your branch. I have a hunch as to what might be happening based on your merge history. Just to check, are you you running `git merge origin/dev` rather than `git merge dev`? The latter merges your local dev branch, so it will only have the changes from your last local pull. The first command is the correct approach to sync with the latest updates from the shared/ remote dev branch. Let me know if you have any questions and I will be happy to explain.

Hi @swoocn ... You're spot on, I suspect this will help moving forward. Thanks for the clarification.

**Comment by swoocn** [comment]
> > Hi @DarinHajou, I've noticed that recent changes from shared dev seem to be missing in your branch. I have a hunch as to what might be happening based on your merge history. Just to check, are you you running `git merge origin/dev` rather than `git merge dev`? The latter merges your local dev branch, so it will only have the changes from your last local pull. The first command is the correct approach to sync with the latest updates from the shared/ remote dev branch. Let me know if you have any questions and I will be happy to explain.
> 
> > Hi @DarinHajou, I've noticed that recent changes from shared dev seem to be missing in your branch. I have a hunch as to what might be happening based on your merge history. Just to check, are you you running `git merge origin/dev` rather than `git merge dev`? The latter merges your local dev branch, so it will only have the changes from your last local pull. The first command is the correct approach to sync with the latest updates from the shared/ remote dev branch. Let me know if you have any questions and I will be happy to explain.
> 
> Hi @swoocn ... You're spot on, I suspect this will help moving forward. Thanks for the clarification.

Great to hear we figured it out!  That being said, could you please create a new branch from the latest remote/dev and reapply your changes?  Thanks so much in advance!
I'm assuming the same will apply for all your other active branches.. 🤔

**Comment by swoocn** [comment]
This PR is superseded by [https://github.com/map-of-pi/map-of-pi-backend-react/pull/166](https://github.com/map-of-pi/map-of-pi-backend-react/pull/166).
